app/views/requests/request/_umlaut_services.html.erb

```diff
@@ -121,17 +121,17 @@
       </div>
 
         <p id='bd_available' class='alert alert-success' aria-live="polite" style='display: none'>
-          <strong>Good news!</strong> This title is available via Borrow Direct,
+          This title is available via Borrow Direct,
           the fastest way to get an item that is not currently on the shelf. You just need to select a delivery location below.
         </p>
         <p id='bd_not_available' class='alert alert-warning' aria-live="polite" style='display: none'>
-          <strong>Don't worry!</strong> Your best bet is still Borrow Direct, so check that first. If you can't
+          Don't worry, your best bet is still Borrow Direct, so check that first. If you can't
           find it you have some other options in the form below.
         </p>
         <p id='bd_error' class='alert alert-danger' aria-live="polite" style='display: none'></p>
         <p id='campus_available' class='alert alert-info' aria-live="polite" style='display: none'>
-          <strong>There is another copy of this item available at Princeton</strong>
-          <a href="">View other copy.</a>
+          <strong>There is another copy of this item available at Princeton</strong><br/>
+          <a href="">view other copy.</a>
```

Comment:
This one doesn't make sense to me. "View other copy" isn't part of the sentence that precedes and isn't a sentence on its own.